scripts/scripts.js

```diff
@@ -154,7 +154,16 @@ const miloLibs = setLibs(LIBS);
 }());
 
 (async function loadPage() {
-  const { loadArea, loadDelayed, loadLana, setConfig } = await import(`${miloLibs}/utils/utils.js`);
+  const { loadArea, loadDelayed, loadLana, setConfig, createTag } = await import(`${miloLibs}/utils/utils.js`);
+  const metaCta = document.querySelector('meta[name="chat-cta"]');
+  if (metaCta && !document.querySelector('.chat-cta')) {
+    const isMetaCtaDisabled = metaCta?.content === 'off';
+    if (!isMetaCtaDisabled) {
+      const chatDiv = createTag('div', { class: 'chat-cta meta-cta', 'data-content': metaCta.content });
+      const lastSection = document.body.querySelector('main > div:last-of-type');
+      if (lastSection) lastSection.insertAdjacentElement('beforeend', chatDiv);
+    }
+  }
```

Comment:
Nit: seems like you could consolidate
```suggestion
  if (metaCta?.content !== 'off' && !document.querySelector('.chat-cta')) {
    const chatDiv = createTag('div', { class: 'chat-cta meta-cta', 'data-content': metaCta.content });
    const lastSection = document.body.querySelector('main > div:last-of-type');
    if (lastSection) lastSection.insertAdjacentElement('beforeend', chatDiv);
  }
```

Reply:
This caused an error 'cannot read properties of null' which is a bit odd. So reverting to how it was before.